dev/io.openliberty.microprofile.telemetry.1.0.internal_fat/fat/src/io/openliberty/microprofile/telemetry/internal_fat/apps/api/ContextAPIServlet.java

```diff
@@ -90,12 +90,13 @@ public static MyImplicitContextKeyed get(Context context) {
     public void testContextStorage() {
         ContextStorage storage = ContextStorage.get();
         Context context = Context.current();
-        Scope scope = storage.attach(context);
-        assertNotNull(scope);
-        assertNotSame(Scope.noop(), scope);
-        Context stored = storage.current();
-        assertEquals(context, stored);
-        scope.close();
+        try(Scope scope = storage.attach(context)){
+            assertNotNull(scope);
+            assertNotSame(Scope.noop(), scope);
+            Context stored = storage.current();
+            assertEquals(context, stored);
+            scope.close();
+        }
```

Comment:
`scope.close()` not needed when using try-with-resources

The formatting around the `try` line doesn't look like it has spaces in the right places.